dapp/.env.example

```diff
@@ -1,34 +1,32 @@
+# Network
 VITE_USE_TESTNET=true
 
-# Configuration of sentry.io
-VITE_SENTRY_SUPPORT=false
-# TODO: Sentry DSN will be added during the application building process when it is ready
-VITE_SENTRY_DSN=""
-
-# TODO: Use a more general source
-VITE_ETH_HOSTNAME_HTTP="https://sepolia.infura.io/v3/c80e8ccdcc4c4a809bce4fc165310617"
+# Basic UI settings
 VITE_REFERRAL=0
 
-# ENDPOINTS
-VITE_TBTC_API_ENDPOINT=""
+# Endpoints
+VITE_ETH_HOSTNAME_HTTP=""
 VITE_ACRE_API_ENDPOINT="http://localhost:8788/api/v1/"
+VITE_TBTC_API_ENDPOINT="http://localhost:8788/tbtc-api/v1/"
 
-# API KEYS
+# API keys
 VITE_GELATO_RELAY_API_KEY="htaJCy_XHj8WsE3w53WBMurfySDtjLP_TrNPPa6IPIc_"  # this key should not be used on production
-# Get the API key from: https://thegraph.com/studio/apikeys/.
 VITE_SUBGRAPH_API_KEY=""
 
+# Sentry
+VITE_SENTRY_SUPPORT=false
+VITE_SENTRY_DSN=""
+
 # Posthog
 VITE_POSTHOG_API_HOST="https://us.i.posthog.com"
 VITE_POSTHOG_API_KEY=""
 
 # Feature flags
-VITE_FEATURE_FLAG_WITHDRAWALS_ENABLED="false"
-VITE_FEATURE_FLAG_OKX_WALLET_ENABLED="false"
-VITE_FEATURE_FLAG_XVERSE_WALLET_ENABLED="false"
+VITE_FEATURE_FLAG_WITHDRAWALS_ENABLED="true"
+VITE_FEATURE_FLAG_OKX_WALLET_ENABLED="true"
+VITE_FEATURE_FLAG_XVERSE_WALLET_ENABLED="true"
 VITE_FEATURE_FLAG_ACRE_POINTS_ENABLED="true"
 VITE_FEATURE_FLAG_TVL_ENABLED="true"
 VITE_FEATURE_GATING_DAPP_ENABLED="true"
 VITE_FEATURE_POSTHOG_ENABLED="true"
```

Comment:
I would change it to `false` by default. We do not use the posthog on development.

Reply:
https://github.com/thesis/acre/pull/921/commits/795814037f0c2c5e35a0cf722e00231fec068ab5